<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TankDialog</class>
 <widget class="QDialog" name="TankDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="top_group">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#top_group {
background-image: url(:/Tank.Platform.Qt/bg.png); 
border: none;
border-bottom:1px solid #606161
}

</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="tank_logo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/Tank.Platform.Qt/tank_logo.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">/* want this stylesheet to apply to the label but not the tooltip */
QLabel{
color: white;
font-size: 20px;
margin-left: 5px;
}</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_context">
            <property name="toolTip">
             <string>foo bar</string>
            </property>
            <property name="styleSheet">
             <string notr="true">/* want this stylesheet to apply to the label but not the tooltip */
QLabel { color: rgba(250,250,250,180);
font-size: 11px;
margin-right: 8px;
} 


</string>
            </property>
            <property name="text">
             <string>Current Work Area:
TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="details">
            <property name="minimumSize">
             <size>
              <width>34</width>
              <height>34</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="toolTip">
             <string>Click for App Details</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
width: 12px;
height: 20px;
background-image: url(:/Tank.Platform.Qt/arrow.png);
border: none;
background-color: none;
}


QToolButton:hover{
background-image: url(:/Tank.Platform.Qt/arrow_hover.png);
}

QToolButton:Pressed {
background-image: url(:/Tank.Platform.Qt/arrow_pressed.png);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="target">
         <property name="spacing">
          <number>4</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QGroupBox" name="page_2_group">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>145</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gradient">
            <property name="minimumSize">
             <size>
              <width>11</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>11</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#gradient {
background-image: url(:/Tank.Platform.Qt/gradient.png); 
border: none;
}</string>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*
All labels inside this scroll area should be 12px font.
This is to avoid the UI looking different in different app like
maya and nuke which all use slightly different style sheets.
 */
QLabel{
   font-size: 11px;
   margin-bottom: 8px
}
</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>383</width>
               <height>566</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="app_icon">
                  <property name="minimumSize">
                   <size>
                    <width>64</width>
                    <height>64</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>64</width>
                    <height>64</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="resources.qrc">:/Tank.Platform.Qt/default_app_icon_256.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="spacing">
                   <number>1</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="app_name">
                    <property name="styleSheet">
                     <string notr="true">font-size: 16px;
</string>
                    </property>
                    <property name="text">
                     <string>Publish And Snapshot</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="app_description">
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Tools to see what is out of date in your scene etc etc.</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="app_tech_details">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="text">
                 <string>tk-multi-snapshot, v1.2.3</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QToolButton" name="btn_documentation">
                  <property name="text">
                   <string>Documentation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btn_support">
                  <property name="text">
                   <string>Help &amp;&amp; Support</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">font-size: 16px;
margin-top: 30px;</string>
                </property>
                <property name="text">
                 <string>Your Current Work Area</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="app_work_area_info">
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QToolButton" name="btn_file_system">
                  <property name="text">
                   <string>Jump to File System</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btn_shotgun">
                  <property name="text">
                   <string>Jump to Shotgun</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="app_work_area_info_2">
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>If you are making changes to configuration or code, use the reload button to quickly load your changes in without having to restart:</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="btn_reload">
                <property name="text">
                 <string>Reload Engine and Apps</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="config_header">
                <property name="styleSheet">
                 <string notr="true">font-size: 16px;
margin-top: 30px;</string>
                </property>
                <property name="text">
                 <string>Configuration</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="config_line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="config_label">
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Below is a list of all the configuration settings for this app, as defined in your environment file:</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="config_layout">
                <property name="spacing">
                 <number>20</number>
                </property>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>328</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
